src/components/Button/Button.js

```diff
@@ -0,0 +1,36 @@
+import { h } from 'preact';
+import Icon from 'react-fontawesome';
+
+const Button = ({ icon, text, disabled, onClick }) => (
+    <button onClick={onClick} disabled={disabled}>
+        { icon ? <Icon name={icon} /> : null }
```

Comment:
How to control icon position (left/right)?

Reply:
Do we need it?

Reply:
Check out your comment in a linked issue :)